crowdin.yml

```diff
@@ -0,0 +1,51 @@
+#
+# Your Crowdin credentials
+#  - The _env versions replace the non-_env versions when using a public repository for security
+#  - These environment variables must be defined in the env: section in your GitHub workflow
+#
+"project_id_env": CROWDIN_PROJECT_ID
+"api_token_env": CROWDIN_API_TOKEN
+"base_path" : "."
+"base_url" : "https://api.crowdin.com"
+
+#
+# Whether to use the existing file system hierarchy or to flatten it out
+#  - true or false, SMF needs this to be true, since our hierarchy isn't flat (agreement.txt vs others)
+#
+"preserve_hierarchy": true
+
+#
+# Files configuration
+#
+files: [
+ {
+  #
+  # Source files filter
+  #  - Grab all the english files in the languages folder
+  #
+  "source" : "/Themes/default/languages/*.english.php",
+
+  #
+  # Where translations will be placed
+  #  - Note this honors CrowdIn's "Language Mapping"
+  #  - Need to remove the '.english' from the filename on export otherwise things end up like "Admin.english.dutch.php"
+  #
+  "translation" : "/Themes/default/languages/%file_name%.%language%.php",
+  "translation_replace": {
+      ".english" : ""
+  }
+ },
+ {
+  #
+  # Source files filter
+  #  - The english agreement has no language in the name
+  #
+  "source" : "agreement.txt",
+
+  #
+  # Translated agreements include the language
+  #
+  "translation" : "agreement.%language%.txt"
+ }
+
+]
```

Comment:
I made this to look more yaml and  less like json
```suggestion
#
# Your Crowdin credentials
#  - The _env versions replace the non-_env versions when using a public repository for security
#  - These environment variables must be defined in the env: section in your GitHub workflow
#
project_id_env: CROWDIN_PROJECT_ID
api_token_env: CROWDIN_API_TOKEN
base_path: "."
base_url: "https://api.crowdin.com"

#
# Whether to use the existing file system hierarchy or to flatten it out
#  - true or false, SMF needs this to be true, since our hierarchy isn't flat (agreement.txt vs others)
#
preserve_hierarchy: true

#
# Files configuration
#
files:
  -
   #
   # Source files filter
   #  - Grab all the english files in the languages folder
   #
   source: /Themes/default/languages/*.english.php

   #
   # Where translations will be placed
   #  - Note this honors CrowdIn's "Language Mapping"
   #  - Need to remove the '.english' from the filename on export otherwise things end up like "Admin.english.dutch.php"
   #
   translation: /Themes/default/languages/%file_name%.%language%.php
   translation_replace:
    .english: ""

  -
   #
   # Source files filter
   #  - The english agreement has no language in the name
   #
   source: agreement.txt

   #
   # Translated agreements include the language
   #
   translation: agreement.%language%.txt
```

Reply:
The JSON-ish structure is required here, because we have two sets of sources & translations.  This is because agreement.txt & the rest behave very differently, as described in the comments above.

Reply:
I started with their templates...  The BIGGEST thing that bugs me is the fact they used spaces, not tabs...

Reply:
FYI - this file is the output of their config tool....

Reply:
in the end it doesn't really matter because both versions are valid and they both output the  exact same results (json)